x/ibc/04-channel/types/tests/types.go

```diff
@@ -0,0 +1,168 @@
+package channel
```

Comment:
Seems like some of this could be combined with test code in ICS 2.

Reply:
yeah... need to redo all the tests since they all rely on the store accessors